TestingHelpers/MockFileSystem.cs

```diff
@@ -0,0 +1,249 @@
+﻿using System.Collections.Generic;
```

Comment:
Something went wrong with the rebase. This file was moved to `System.IO.Abstractions.TestingHelpers/MockFileSystem.cs`.

Can you please fix this? Otherwise it's pretty hard to review what has changed here.

Thanks!

Reply:
@russcam Any chance to fix this?